packages/astro/package.json

```diff
@@ -4,6 +4,7 @@
   "author": "Skypack",
   "license": "MIT",
   "type": "module",
+  "types": "./dist/types/@types/astro-core.d.ts",
```

Comment:
In the distributed version of `astro-core.d.ts`, the [reference to the Vite path](https://github.com/snowpackjs/astro/blob/main/packages/astro/src/%40types/astro-core.ts#L5) is incorrect:

```diff
- import type vite from '../../vendor/vite';
+ import type vite from '../../../vendor/vite';
```

This is complicated, tho. The current path is correct in the source. It is only incorrect in its distributed path.